drupal/web/themes/custom/camp/templates/content/node--article--full.html.twig

```diff
@@ -94,6 +94,7 @@
         {{ label }}
       </h1>
     {{ title_suffix }}
+    <p class="typo-italic last-updated--timestamp">{{ 'Last updated @updated_timestamp'|trans({'@updated_timestamp': updated_timestamp}) }}</p>
```

Comment:
It seems that it is possible for updated_timestamp to not be set in which case this line would just end up being rendered as "Last updated (blank)", or am I missing something?

Reply:
AAAAH! how I didn't notice that, I can't say. Will fix.